modules/nw-multus-add-pod.adoc

```diff
@@ -13,11 +13,18 @@ endif::[]
 You can add a Pod to an additional network. The Pod continues to send normal
 cluster related network traffic over the default network.
 
+If a Pod already exists, you cannot attach it to an additional network. Only when a Pod is created are any additional networks requested attached to the Pod.
+
 ifdef::sriov[]
 [NOTE]
 =====
-The Network Resources Injector will inject the `resource` parameter into the Pod CR automatically if a NetworkAttachmentDefinition CR associated with the SR-IOV CNI plug-in is specified.
+The Network Resources Injector will inject the `resource` field into the Pod custom resource (CR) automatically if a NetworkAttachmentDefinition CR associated with the SR-IOV Container Network Interface (CNI) plug-in is specified.
 =====
+
+[IMPORTANT]
+====
+When specifying an SR-IOV hardware network, you must specify the namespace of the NetworkAttachmentDefinition CR. For more information, see the following bugs: link:https://bugzilla.redhat.com/show_bug.cgi?id=1846333[BZ#1846333] and link:https://bugzilla.redhat.com/show_bug.cgi?id=1840962[BZ#1840962].
```

Comment:
@zshi-redhat, so I can remove this reference to these bugs, then?

Reply:
These two bugs were fixed in 4.6, we should remove them for 4.6 doc.
It should be added back when backporting to 4.5 doc or earlier releases (because the bug fix is not cherry-pick to 4.5 or earlier releases).

Reply:
@zshi-redhat, okay, I've added a conditional so that these bugs appear only in 4.3, 4.4, and 4.5.

Reply:
@jboxman the fix looks good to me.

nit: since the issue only happens when sriov network is specified in a deployment/replicacontroller, shall we 
change:
`when specifying an SR-IOV hardware network`
to:
`when specifying an SR-IOV hardware network for a deployment or replicacontroller` ?

Reply:
@zshi-redhat, updated, thanks!